ajax/libs/iphone-inline-video/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "iphone-inline-video",
+  "filename": "iphone-inline-video.min.js",
+  "version": "2.2.2",
+  "description": "Make videos playable inline on the iPhone (prevents automatic fullscreen)",
+  "homepage": "https://bfred-it.github.io/iphone-inline-video/",
```

Comment:
Oh, let's use `http://bfred-it.github.io/iphone-inline-video/` not `https`
Because the SSL issue.
Thanks.

Reply:
Again, the ssl is working for me, But I will change it to http.

Reply:
Ummmm....
In fact, I also can access `https`.
But there is no green lock in my chrome browser.
However, in this case I don't know is it good to use `http`.
Because there is no problem in certification: https://www.ssllabs.com/ssltest/analyze.html?d=bfred-it.github.io&s=151.101.53.147&latest

Maybe we can keep this part this time.
But I not sure.